bindings/wysiwyg-wasm/src/lib.rs

```diff
@@ -457,7 +463,7 @@ impl DomChildren {
     }
 
     // Clippy suggests that this name is ambiguous
-    pub fn next(&mut self) -> Option<DomHandle> {
+    pub fn next_child(&mut self) -> Option<DomHandle> {
```

Comment:
Probably the only change with the potential for knock on effects. Could only find a single usage which is renamed in platforms/web/lib/dom.ts